### PR TITLE
Update setup.py with latest details

### PR DESCRIPTION
**Pull request by dimon222**
Some minor corrections.
This library is no more just plain beta (so respective classifier got removed) + repo got moved to new location, so I updated url.

I don't know if we should touch author's email, I might assume in Pypi it plays a role of way to contact author (?), tho, I don't know if @toidi  is still reachable via it anymore. Even if we decide to change it, don't know what to put there.

### CODE REVIEW

**Comment by toidi** [comment]
Hey @dimon222 ,

Thanks for the update!
That email is no longer active. Feel free to remove/change it.

**Comment by dimon222** [comment]
@lresende is there some CODAIT contact mailbox that we could plug there?

**Comment by lresende** [comment]
We don't have anything specific, but most of us have public e-mails on our profiles but we should always encourage communication via issues.

**Comment by dimon222** [comment]
@lresende I see. I have done some reading, in our case we could utilize maintainer & maintainer_email. I've added myself there as last point of contact (but I would assume anyway most people will appear in this repo in "Issues")

File: setup.py [hunk 1]
```diff
@@ -42,10 +42,11 @@ def find_version(*file_paths):
 
     author = 'Iskandarov Eduard',
     author_email = 'e.iskandarov@corp.mail.ru',
```
Comment (kevin-bates) [inline setup.py:44]: Given @toidi said this email is no longer active, I would suggest it be removed.

**Review by kevin-bates — APPROVED** [review]
Thanks Dmitry.

**Review by lresende — CHANGES_REQUESTED** [review]
I have some suggestions and will provide it later today or early tomorrow.

File: setup.py [hunk 2]
```diff
@@ -41,11 +41,12 @@ def find_version(*file_paths):
     test_suite = 'tests',
 
     author = 'Iskandarov Eduard',
-    author_email = 'e.iskandarov@corp.mail.ru',
+    author_email = None,
```
Comment (toidi) [inline setup.py:44]: in case you retain author_email parameter - you could use my personal email: `eduard.iskandarov@ya.ru`

**Comment by dimon222** [comment]
@lresende got any update?

**Review by lresende — APPROVED** [review]

**Pull request merged**